nomos-mix/core/src/message_blend/crypto.rs

```diff
@@ -4,7 +4,7 @@ use serde::{Deserialize, Serialize};
 /// [`CryptographicProcessor`] is responsible for wrapping and unwrapping messages
 /// for the message indistinguishability.
 #[derive(Clone, Copy, Debug)]
-pub(crate) struct CryptographicProcessor {
+pub struct CryptographicProcessor {
```

Comment:
This is used directly now for encoding the local messages, its much more simple!!